pkg/operation/botanist/component/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -553,31 +556,56 @@ func (k *kubeControllerManager) computeCommand(port int32) []string {
 	var (
 		command                              []string
 		defaultHorizontalPodAutoscalerConfig = k.getHorizontalPodAutoscalerConfig()
+		podEvictionTimeout                   = metav1.Duration{Duration: 2 * time.Minute}
+		nodeMonitorGracePeriod               = metav1.Duration{Duration: 2 * time.Minute}
 	)
 
 	command = append(command,
 		"/usr/local/bin/kube-controller-manager",
-		"--allocate-node-cidrs=true",
 		"--attach-detach-reconcile-sync-period=1m0s",
```

Comment:
Why is this flag still needed in workerless scenario?

Reply:
It is not needed and is only added for shoots with workers.